docs/ref/migration-operations.txt

```diff
@@ -37,7 +37,9 @@ Schema Operations
 ``CreateModel``
 ---------------
 
-.. class:: CreateModel(name, fields, options=None, bases=None, managers=None)
+..versionchanged:: 3.2
+
+.. class:: CreateModel(name, fields, options=None, bases=None, managers=None, metaclass=None)
```

Comment:
I'd drop this change. I think it is covered under the note lower down.

Reply:
The `..versionchanged:: 3.2` can be dropped here, but the added `metaclass=None` is necessary.